test/libscap/test_suites/engines/savefile/converter.cpp

```diff
@@ -184,3 +186,49 @@ TEST_F(convert_event_test, PPME_SYSCALL_PREAD_X__to_4_params_with_enter) {
 	                               size,
 	                               pos));
 }
+
+////////////////////////////
+// BIND
+////////////////////////////
+
+TEST_F(convert_event_test, PPME_SOCKET_BIND_E_store) {
+	uint64_t ts = 12;
+	int64_t tid = 25;
+
+	int64_t fd = 25;
+	uint32_t size = 89;
+
+	auto evt = create_safe_scap_event(ts, tid, PPME_SOCKET_BIND_E, 1, fd, size);
+	assert_single_conversion_skip(evt);
+	assert_event_storage_presence(evt);
+}
+
+TEST_F(convert_event_test, PPME_SOCKET_BIND_X_to_3_params_no_enter) {
```

Comment:
if we want to reduce the number of tests I would test the case with enter rather than the case without it, or we can keep both. In any case i would add the test enter+exit

Reply:
Ok, I think it's better to just add the "with_enter" test, without removing the "no_enter" one. I'll do it.